src/main/java/com/alvarium/annotators/ScaAnnotator.java

```diff
@@ -3,19 +3,20 @@
 import java.net.InetAddress;
 import java.net.UnknownHostException;
 import java.time.Instant;
+import java.util.List;
```

Comment:
We want the copyright header at the top of every touched file in this PR, you'll find it at the top of other files so its safe to copy and paste but make sure to update the date to 2023.